packages/core/src/fields/types/checkbox/views/index.tsx

```diff
@@ -17,16 +17,19 @@ export const Field = ({ field, value, onChange, autoFocus }: FieldProps<typeof c
   return (
     <FieldContainer>
       <Checkbox
+        css={{ alignItems: 'center' }}
```

Comment:
We need the checkbox to align with the top of the field label with/without the description
![image](https://user-images.githubusercontent.com/6447754/172544967-2f1e5fdd-be37-4cdc-82f8-3cb72add7398.png)

Reply:
Also, the whitespace between label/description needs to match other fields
![image](https://user-images.githubusercontent.com/6447754/172547244-3c220983-97bf-472d-b950-0360e81786c4.png)

Reply:
Resolved the styling for Checkbox